src/man/firejail.txt

```diff
@@ -2029,6 +2029,14 @@ For details, see
 #903
 .UE
 
+.TP
+\fB\-\-noskel
+.br
+By default, when using a private home directory, firejail creates files based
+on the operating system home "skeleton". (i.e. ~/.bashrc, ~/.bash_profile, ...).
+This option disabled this feature, and enables the user to whitelist these
```

Comment:
Use present tense? This option `disables` this ...

Reply:
Rha. Don't write pull request before going to bed :) . Good point.